yaml-tests/yaml-tests.gradle

```diff
@@ -73,7 +69,58 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junitVersion}"
 }
 
+ext.resolveOtherServer = { ->
+    def configurationName = "resolveArtifact-${UUID.randomUUID()}"
+    return rootProject.with {
+        configurations.create(configurationName, {
+            resolutionStrategy {
+                failOnChangingVersions()
+                // Note: In the root of the project we set cacheChangingModulesFor and cacheDynamicVersionFor to 0
+                // We explicitly change that for changing modules, because we should reject those anyways.
+                cacheChangingModulesFor 1000, "days" // TTL for snapshots
```

Comment:
This comment may already be out-of-date, at least I couldn't find any references to `cacheChangingModulesFor` in the repo, other than this one.

Reply:
Let's fix that in a subsequent PR.